spyder/widgets/editor.py

```diff
@@ -274,6 +275,119 @@ def breakpoints_changed(self):
             self.save_breakpoints.emit(self.filename, repr(breakpoints))
 
 
+class StackHistory(MutableSequence):

```

Comment:
Sorry, don't worry about it. We simply need to split this file in multiple files because it's huge!!

But let's do that in `master` (I'll do it).